Core/include/Acts/Surfaces/TrapezoidBounds.hpp

```diff
@@ -54,14 +55,29 @@ class TrapezoidBounds : public PlanarBounds {
     checkConsistency();
   }
 
+  /// Constructor for symmetric Trapezoid with rotation
+  ///
+  /// @param halfXnegY minimal half length X, definition at negative Y
+  /// @param halfXposY maximal half length X, definition at positive Y
+  /// @param halfY half length Y - defined at x=0
+  /// @param rotAngle: rotation angle of the bounds w.r.t coordinate axes
+  TrapezoidBounds(double halfXnegY, double halfXposY, double halfY,
+                  double rotAngle) noexcept(false)
```

Comment:
I'm wondering if adding the rotation angle as an argument to the existing constructor and defaulting it to zero might be nicer since it shows the default more explicitly.

Reply:
No real preference

Reply:
Then let's use a defaulted argument.